src/pages/example-1/Example1.tsx

```diff
@@ -1,38 +1,46 @@
 import React from 'react';
 import './Example1.css';
 import Table from '../../components/table/Table';
-import { Certificate } from '../../types/types';
+import { Certificate } from '@/types/types';
+import { useNavigate } from 'react-router-dom';
+
+export const certificates: Certificate[] = [
```

Comment:
Please move this array into separate file, for example certificate-mock-data.ts, it can stay in same folder like this component

Reply:
not resolved yet

Reply:
not resolved yet